pkg/upload/dsym.go

```diff
@@ -146,14 +147,14 @@ func ProcessDsym(
 			if apiKey == "" {
 				apiKey = plistData.BugsnagProjectDetails.ApiKey
 				if apiKey != "" {
-					log.Info(fmt.Sprintf("Using API key from Info.plist: %s", apiKey))
+					logger.Debug(fmt.Sprintf("Using API key from Info.plist: %s", apiKey))
 				}
 			}
 		}
 
 		for _, dsym := range dwarfInfo {
 			dsymInfo := fmt.Sprintf("(UUID: %s, Name: %s, Arch: %s)", dsym.UUID, dsym.Name, dsym.Arch)
-			log.Info(fmt.Sprintf("Uploading dSYM %s", dsymInfo))
+			logger.Info(fmt.Sprintf("Uploading dSYM %s", dsymInfo))
```

Comment:
Given the "Uploading ..." message is emitted from `ProcessFileRequest`, perhaps make this debug and the message a bit different?

```suggestion
			logger.Debug(fmt.Sprintf("Processing dSYM %s", dsymInfo))
```